src/docs/sdk/performance/dynamic-sampling-context.mdx

```diff
@@ -0,0 +1,158 @@
+To propagate DSC to downstream SDKs/services, we create a baggage header (or modify an existing one) through HTTP request instrumentation.
+
+<Alert level="info">
+
+Other vendors might also be using the `baggage` header.
+If a `baggage` header already exists on an outgoing request, SDKs should aim to be good citizens by only **appending** Sentry values to the header.
+In the case that another vendor added Sentry values to an outgoing request, SDKs may overwrite those values.
+
+SDKs must not add other vendors' baggage from incoming requests to outgoing requests.
+Sentry SDKs only concern themselves with Sentry baggage.
+
+</Alert>
+
+The following is an example of what a baggage header containing Dynamic Sampling Context may look like:
+
+```
+baggage: other-vendor-value-1=foo;bar;baz, sentry-traceid=771a43a4192642f0b136d5159a501700, sentry-publickey=49d0f7386ad645858ae85020e393bef3; sentry-userid=Am%C3%A9lie, other-vendor-value-2=foo;bar;
+```
+
+See the [Payloads section](#payloads) for a complete list of key-value pairs that SDKs should propagate.
+
+## Payloads
+
+Dynamic Sampling Context is propagated via a baggage header and sent to Sentry via <Link to="/sdk/envelopes/#transaction">transaction envelope headers</Link>.
+
+### Baggage-Header
+
+SDKs may set the following key-value pairs on baggage headers.
+While all of these values are optional, SDKs should make their best effort to add as many of them to the baggage header as possible when starting a trace.
+
+- `sentry-traceid` - The original trace ID as generated by the SDK
+- `sentry-publickey` - Public key as defined by the user via the DSN in the SDK options
+- `sentry-release` - The release as defined by the user in the SDK options
+- `sentry-environment` - The environment as defined by the user in the SDK options
+- `sentry-transaction` - The name of the trace's origin transaction in unparameterized (raw) format
+- `sentry-userid` - User ID as set by the user with <Link to="/sdk/unified-api/#scope">`scope.set_user`</Link>
+- `sentry-usersegment` - User segment as set by the user with <Link to="/sdk/unified-api/#scope">`scope.set_user`</Link>
+- `sentry-samplerate` - Sample rate as defined by the user in the SDK options
+
+SDKs must set all of the keys in the form of "`sentry-[name]`".
+The delimiter "`sentry-`" acts to identify key-value pairs set by Sentry SDKs.
+Additionally, we chose `[name]` to be written in "snake case" without any underscore ( `_` ) characters. This naming convention is the most language agnostic.
+
+### Envelope Header
+
+Dynamic Sampling Context is transferred to Sentry through the <Link to="/sdk/envelopes/#transaction">transaction envelope headers</Link>, keyed by `trace`.
+It corresponds directly to the definition of <Link to="/sdk/performance/trace-context/#trace-context">Trace Context</Link>.
+
+When a transaction is reported to Sentry, the Dynamic Sampling Context must be mapped to Trace Context in the following way:
```

Comment:
I thought the `sentry-` prefix was for baggage and we wanted to rename Trace Context to Dynamic Sampling Context?

Reply:
Kinda answered this already somewhere else: The term `Dynamic Sampling Context` refers to `sentry-*` values in the baggage header. Trace Context is something different (object schema in transaction envelope header).

Reply:
I'll resolve this because the [discussion](https://github.com/getsentry/develop/pull/613#discussion_r903583558) about this happened in another thread in this PR. Feel free to reopen if necessary